test/net/sourceforge/kolmafia/textui/command/SendMessageCommandTest.java

```diff
@@ -0,0 +1,27 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+import org.junit.jupiter.api.Test;
+
+class SendMessageCommandTest extends AbstractCommandTestBase {
+
+  public SendMessageCommandTest() {
+    this.command = "csend";
+  }
+
+  @Test
+  public void itShouldParseMeatWithCommas() {
+    String output = execute(" 1,000,000 meat to buffy");
```

Comment:
this should send no meat and just error, I think. we don't support numbers with comments

Reply:
The problem with these tests is what I have written only captures that a message was sent and not the contents of the message.  If you can point me at how to get the message I will gladly expand.  Alternatively I can shift the focus to increasing coverage according to jacoco in which case deleting this test makes sense.